org.messerlab.slimgui.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.messerlab.slimgui
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=com.canonical.AppMenu.Registrar # required for global menu
+  - --talk-name=org.freedesktop.Flatpak # allows spawning processes on the host via flatpak-spawn --host
+  - --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig
+command: SLiMgui
+modules:
+  - name: SLiMgui
+    builddir: true
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_SLIMGUI=ON
+    sources:
+      - type: git
+        url: https://github.com/messerlab/slim.git
+        tag: v4.0.1
+        commit: 09fcf604a06307cf75098dfa0a2171a9b7852216
+    post-install:
+      # Colourful and Symbolic Application Icons
+      # This file is modified to produce a symbolic icon that works well with GNOME 3; the changes haven't been integrated upstream yet.
+      - sed -i '69,166 s/"fill:#[[:alnum:]]*"/"fill:#ffffff"/' ../QtSLiM/icons/AppIcon16.svg
+      - sed -i '62,68 d' ../QtSLiM/icons/AppIcon16.svg
+      - install -Dm644 ../QtSLiM/icons/AppIcon64.svg /app/share/icons/hicolor/scalable/apps/org.messerlab.slimgui.svg
+      - install -Dm644 ../QtSLiM/icons/AppIcon16.svg /app/share/icons/hicolor/symbolic/apps/org.messerlab.slimgui-symbolic.svg
+
+      # slim Mimetype
+      # Line 6 in original org.messerlab.slimgui-mime.xml
+      # <generic-icon name="/usr/share/icons/hicolor/scalable/mimetypes/text-slim.svg"/>
+      - sed -i '6 s,/usr/share/icons/hicolor/scalable/mimetypes/text-slim.svg,org.messerlab.slimgui.text-slim.svg,' ../org.messerlab.slimgui-mime.xml
+      - install -Dm644 ../org.messerlab.slimgui-mime.xml /app/share/mime/packages/org.messerlab.slimgui.xml
+      - install -Dm644 ../QtSLiM/icons/DocIcon.svg /app/extra/export/share/icons/hicolor/scalable/mimetypes/org.messerlab.slimgui.text-slim.svg
+
+      # Desktop Entry and Appstream XMLs
```

Comment:
these should be installed by upstream (next release?)

Reply:
That can be done. I'll open a PR against our cmakelist

Reply:
> these should be installed by upstream (next release?)

Were this to be included in the upstream `CMakeLists.txt` installation section, would `flatpak-builder` still expect the mimetype icon to be named `org.messerlab.slimgui.text-slim.svg`? When I created the mimetypes I spent significant time reading the FreeDesktop specifications to ensure I was writing things correctly. If you can point me to the correct information that would be helpful. If I make the change upstream with the Flatpak "-style" icon naming, might it break other package formats desktop integration?